### DISCUSSION TITLE
Can I join team?

### DISCUSSION THREAD

**Discussion by rahulkhatri137** [ROM-builders/temporary#11844] [Q&A]
Successful builds -

* Xdroid - https://cirrus-ci.com/build/5452528669163520
* CipherOS - https://cirrus-ci.com/build/5573831652802560
* RiceDroid - https://cirrus-ci.com/build/5122240449609728
* RiceDroid 2- https://cirrus-ci.com/build/5315268896358400

Info -
* Github - https://github.com/rahulkhatri137 (main account) &
https://github.com/parkerpet (used to create PR for builds)
* Telegram - https://t.me/rahulkhatri137

Thanks for helping us with free machine

**Comment by Apon77** [comment]
please follow #learn instruction

**Reply by rahulkhatri137** [reply]
Already read those
I know git & maintaining my device tree now

**Reply by rahulkhatri137** [reply]
@Apon77 Do I need build more or 4 builds enough for approval?

**Reply by Apon77** [reply]
Your profile does not say that

**Reply by rahulkhatri137** [reply]
I actually built from two profiles 🙂 I am using new account now bc this account has been suspended by cirrus 🙂
It's @parkerpet now. So I have two builds on each account.
I have linked the builds
